Dockerfile

```diff
@@ -37,9 +53,8 @@ COPY --from=operator-build /usr/lib/libgpgme.so.* /usr/lib/
 COPY --from=k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.1 /csi-node-driver-registrar /usr/local/bin
 COPY --from=k8s.gcr.io/sig-storage/livenessprobe:v2.7.0 /livenessprobe /usr/local/bin
 
-# csi depdenencies
-COPY --from=dependency-src /bin/mount /bin/umount /bin/
-COPY --from=dependency-src /lib64/libmount.so.1 /lib64/libblkid.so.1 /lib64/libuuid.so.1 /lib64/
+# depdenencies
```

Comment:
we should keep "csi dependencies" and add another section with "general/core/debug dependencies"

Reply:
I'll answer this question as well as the one below:

By specifying the install root in the installation command, the files that are needed are installed in the specified directory.
This means, by default, only the needed binaries and its strong dependencies are in this folder.
There is no difference between copying all contents of this folder at once or copying the specific files one by one, but the additional leg work necessary to list all the files explicitly.
Therefore, I would opt to not list them explicitly which would also mean that there can't be two different sections.

The only thing I actually need to delete is the `os-release` file because that is copied in manually for `microdnf` to work. I'll fix that in the next commit.